app/charts/shared/ChartProps.tsx

```diff
@@ -3,7 +3,8 @@ import { Observation } from "@/domain/data";
 import { DimensionMetadataFragment } from "@/graphql/query-hooks";
 
 export type BaseChartProps = {
-  data: Observation[];
+  chartData: Observation[];
+  scalesData: Observation[];
   dimensions: DimensionMetadataFragment[];
   measures: DimensionMetadataFragment[];
 };
```

Comment:
Here we could have a little `/** comment */` with what you've written in PR description, so that when we are not sure what is scalesData, we can hover over it and see it :)